lib/iris/aux_factory.py

```diff
@@ -105,6 +105,41 @@ def view(self, *args, **kwargs):
         return self._cached_array().view(*args, **kwargs)
 
 
+class LazyArray(_LazyArray):
+    """
+    Represents a simplified NumPy array which is only computed on demand.
+
+    It provides the :meth:`view()` and :meth:`reshape()` methods so it
+    can be used in place of a standard NumPy array under some
+    circumstances.
+
+    The first use of either of these methods causes the array to be
+    computed and cached for any subsequent access.
+
+    .. deprecated:: 2.0
```

Comment:
NB. This might need to change, depending on the outcome of the [group discussion](https://groups.google.com/forum/#!topic/scitools-iris-dev/ZRZyKeVMZSI).